lib/rack/session/encryptor.rb

```diff
@@ -32,12 +32,12 @@ module Serializable
         def serialize_payload(message)
           serialized_data = serializer.dump(message)
 
-          return "#{[0].pack('v')}#{serialized_data}" if @options[:pad_size].nil?
+          return "#{[0].pack('v')}#{serialized_data.force_encoding('BINARY')}" if @options[:pad_size].nil?
```

Comment:
For the sake of cached constant lookup, consider using `Encoding::BINARY` instead.